fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintainer.java

```diff
@@ -503,14 +503,14 @@ private static IndexOptions getIndexOptions(@Nonnull String value) {
     @Override
     public RecordCursor<IndexEntry> scanUniquenessViolations(@Nonnull TupleRange range, @Nullable byte[] continuation, @Nonnull ScanProperties scanProperties) {
         LOG.trace("scanUniquenessViolations");
-        return RecordCursor.empty();
+        return RecordCursor.empty(state.context.getExecutor());
```

Comment:
I suspect it's always the same, but it should use the executor provided to the maintainer, rather than the one on the context.
```suggestion
        return RecordCursor.empty(executor);
```